README.md

```diff
@@ -54,9 +60,109 @@ Kinvey CLI is distributed as an NPM package. After you install NPM, run the foll
 
    Deletes the specified profile or the active one if you don't specify a profile name.
 
+* `org list`
+
+    Lists all existing organizations within your Kinvey account.
+    
+* `org show [org]`
+
+    Shows detailed information about the specified organization or about the active one if you don't specify an organization. You can specify an organization by ID or name.
+
+* `org use <org>`
+
+    Sets the specified organization as active.
+    
+* `app create <name>`
+
+    Creates an application.
+    
+* `app list`
+
+    Lists all existing applications within your Kinvey account.
+    
+* `app show [app]`
+
+    Shows detailed information about the specified application or about the active one if you don't specify an application. You can specify an application by ID or name.
+
+* `app use <app>`
+
+    Sets the specified application as active.
+    
+* `app delete [app]`
+
+    Deletes the specified application or the active one if you don't specify an application. You can specify an application by ID or name.
+
+* `env create <name>`
+
+    Creates an environment within the active application. To use a different application, specify it using `--app`.
+    
+    * `--app <application>`
+       
+      Specifies a Kinvey app by ID or name. Requires `--env`.
+    
+* `env show [env]`
+
+    Shows detailed information about the specified environment or about the active one if you don't specify an environment. You can specify an environment by ID or name. By default, the command searches inside the active application but you can specify a different application using `--app`.
+    
+    * `--app <application>`
+       
+      Specifies a Kinvey app by ID or name. Requires `--env`.
```

Comment:
Actually, it requires `--env` in the collections context. Here, the app only is enough. The same applies for the entire namespace.

Reply:
Thanks, that's a copying-and-pasting error.